app/src/main/java/com/olayg/shibeonline/adapter/ShibeAdapter.kt

```diff
@@ -37,6 +38,7 @@ class ShibeAdapter : RecyclerView.Adapter<ShibeAdapter.ShibeViewHolder>() {
         fun loadUrl(url: String) = with(binding) {
             //ivImage.load(url)
             tvUrl.text = url
+            Glide.with(root).load(url).into(ivImage)
```

Comment:
Create an extension function on the `ImageView` so it has the ability to take a url